sdk/core/core-rest-pipeline/src/constants.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export const SDK_VERSION: string = "1.8.0";
+export const SDK_VERSION: string = "1.9.0";
```

Comment:
should we try to sneak this into 1.8 release this week? It seems pretty low-risk